git-cliff-core/src/changelog.rs

```diff
@@ -835,6 +836,7 @@ mod test {
 				sort_commits:             Some(String::from("oldest")),
 				link_parsers:             None,
 				limit_commits:            None,
+				recurse_submodules:       None,
```

Comment:
not sure whether the name fits, it recurses only with depth 1 but I don't think there is a use case for higher depths.

Reply:
I think that's fine (also sounds cool :D)